apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/LiveAppSDKCallback.ts

```diff
@@ -0,0 +1,357 @@
+import { Dispatch } from "redux";
+import { TFunction } from "react-i18next";
+
+import {
+  Account,
+  CryptoCurrency,
+  Operation,
+  SignedOperation,
+  Transaction,
+} from "@ledgerhq/live-common/types/index";
+import { addPendingOperation, getMainAccount } from "@ledgerhq/live-common/account/index";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { getEnv } from "@ledgerhq/live-common/env";
+import {
+  accountToPlatformAccount,
+  currencyToPlatformCurrency,
+  getPlatformTransactionSignFlowInfos,
+} from "@ledgerhq/live-common/platform/converters";
+import {
+  serializePlatformAccount,
+  deserializePlatformTransaction,
+  serializePlatformSignedTransaction,
+  deserializePlatformSignedTransaction,
+} from "@ledgerhq/live-common/platform/serializers";
+import { AppManifest } from "@ledgerhq/live-common/platform/types";
+import {
+  RawPlatformTransaction,
+  RawPlatformSignedTransaction,
+} from "@ledgerhq/live-common/platform/rawTypes";
+import { ToastData } from "@ledgerhq/live-common/notifications/ToastProvider/types";
+
+import { updateAccountWithUpdater } from "../../actions/accounts";
+import { openModal } from "../../actions/modals";
+
+import * as tracking from "./tracking";
+import { MessageData } from "@ledgerhq/live-common/hw/signMessage/types";
+import { prepareMessageToSign } from "@ledgerhq/live-common/hw/signMessage/index";
+
+type WebPlatformContext = {
+  manifest: AppManifest;
+  dispatch: Dispatch;
+  accounts: Array<Account>;
+};
+
+export const listAccountsCallback = (accounts: Array<Account>) =>
+  accounts.map(account => serializePlatformAccount(accountToPlatformAccount(account)));
+
+export const listCurrenciesCallback = (currencies: Array<CryptoCurrency>) =>
+  currencies.map(currencyToPlatformCurrency);
+
+export const receiveOnAccountCallback = (
+  accountId: string,
+  { manifest, dispatch, accounts }: WebPlatformContext,
```

Comment:
I think we should try to be consistent with the params order and set the context as the first param for every function